frontend/src/constants/trueFalse.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * TrueFalse enum type to ensure strict compile time checking.
+ * Can't be named 'Boolean' because typescript has a type with that name.
+ */
+export enum TrueFalse {
+  True = 'true',
```

Comment:
Use this instead of magic string Boolean values.

Reply:
why does it need to be a string rather than an actual boolean?

Reply:
Good question and one I'd like to know too!  It appears all the Formik forms require the use of string values.  I haven't had opportunity to look into why we don't have a wrapper at that level rather than having to deal with it higher up in the food chain.

Reply:
huh, weird

Reply:
@Fosol can we chat about this enum and its usage. Something does not seem right

Reply:
formik doesn't require all of its values to be strings, we could easily represent boolean flags as booleans as is done elsewhere in the application (such as the edit field in the edit user page). the one caveat is that all booleans require a non-null non-undefined initial value, so either true or false. If that doesn't work, you would need a boolean | '' type with an initial value of ''.

essentially the field itself is responsible for determining what gets saved to formik. In this case it is saving string representations of true and false, which is likely incorrect.